docs/contributors/guide/cherry-pick/cherry-pick.md

```diff
@@ -0,0 +1,129 @@
+  cherry pick.
+- Your cherry pick PR will immediately get the `do-not-merge/cherry-pick-not-approved` label.
+
+## Cherry Pick Review
+
+Unlike other PRs, cherry pick PR will directly copy the LGTMs of the original PR, but we still require `/merge` to
+trigger the merge PR.
+
+The same release note requirements apply as normal pull requests, except the release note stanza will auto-populate from
+the master branch pull request from which the cherry pick originated.
+
+If this is unsuccessful, the `do-not-merge/release-note-label-needed` label will be applied and the cherry pick author
+must edit the pull request description to add a release note or include in a comment the `/release-note-none` command.
+
+Cherry pick pull requests are reviewed slightly differently than normal pull requests on the `master` branch in that
+they:
+
+- Are by default expected to be `type/bug-fix`.
+
+- The original change to the `master` branch is expected to be merged for some time and no related CI failures or test
+  flakiness must be discovered.
+
+- Milestones must be set on the PR reflecting the milestone for the target release branch (for example, milestone v5.3.2
+  for a cherry pick onto branch
+  `release-5.3`). This is normally done for you by manually.
+
+- A separate cherry pick pull request should be open for every applicable target branch. This ensures that the fix will
+  be present on every active branch for a given set of patch releases. If a fix is only applicable to a subset of active
+  branches, it is helpful to note why that is the case on the parent pull request or on the cherry pick pull requests to
+  the applicable branches.
+
+- Have one additional level of review in that they must be approved specifically for cherry pick by branch approvers.
+
+  Approval is signified by an approver manually applying the
+  `cherry-pick-approved` label. This action removes the
+  `do-not-merge/cherry-pick-not-approved` label and triggers a merge into the target branch.
+
+  If you are concerned about the status of your cherry pick, err on the side of overcommunicating and reach out to the
+  [nongfushanquan](https://github.com/nongfushanquan). We should
+  contact [nongfushanquan](https://github.com/nongfushanquan) as early as possible for approval.
+
+## Searching for Cherry Picks
+
+Examples (based on cherry picks targeting the `release-5.3` branch):
+
+- [`cherry-pick-approved`](https://github.com/pingcap/tiflow/pulls?q=is%3Aopen+is%3Apr+label%3Acherry-pick-approved+base%3Arelease-5.3)
+- [`do-not-merge/cherry-pick-not-approved`](https://github.com/pingcap/tiflow/pulls?q=is%3Aopen+is%3Apr+label%3Ado-not-merge%2Fcherry-pick-not-approved+base%3Arelease-5.3)
+
+## Troubleshooting Cherry Picks
+
+Contributors may encounter some of the following difficulties when initiating a cherry pick.
+
+- A cherry pick PR does not apply cleanly against an old release branch. In that case, you will need to manually fix
```

Comment:
in some cases this is caused by parent PR not being merged into release branch (which is waiting approve or forgotten). After the parent PR merged, you can re-trigger the cherry pick from master's PR to see if conflicts are still there

Reply:
Added.

Reply:
Thanks!